### ISSUE TITLE
Consider dropping title above graph

### ISSUE THREAD

**Issue by grtjn** [patrickmcelwee/ml-visjs-graph-ng#26]
Text should be fully configurable, not just partially, and preferably title should be hide-able altogether.

Perhaps add a title option on the directive. If omitted you get old behavior. If provided it replaced original 'Link Analysis...' text with title. If provided but empty, hide title altogether.

**Comment by grtjn** [comment]
I think it has already been removed in latest.. :)

**Comment by patrickmcelwee** [comment]
yes, I removed it ... I don't think it needs to be part of this library. And it was buggy. Agree?

**Comment by grtjn** [comment]
Not sure i would call it buggy, or do you mean the value of ctrl.label? Most graphical components support some kind of title, but the one based on ctrl.label probably wasn't the best.. :)

**Comment by patrickmcelwee** [comment]
Our implementation was buggy. In cases where several nodes came in initially, there was randomness in WHICH label was used for the title. I agree that it could be supported, but I also think users can make their own title quick easily, so I personally don't think it's top priority.

**Comment by grtjn** [comment]
yeah, a low prio nice to have for sure..
